lib/lgy/service.rb

```diff
@@ -51,6 +51,14 @@ def coe_status
         )
         nil
       end
+    rescue Common::Client::Errors::ClientError => e
+      log_message_to_sentry(
+        "GET coe_status failed with http status: #{e.status}",
+        :error,
+        { message: e.message, status: e.status, body: e.body, has_edipi: @edipi.present? },
+        { team: 'vfs-ebenefits' }
```

Comment:
Hey I created our Github team - https://github.com/orgs/department-of-veterans-affairs/teams/benefits-non-disability
Do we want to add it here?

Reply:
no, this is correct as vfs-ebenefits. it is describing the project in `sentry`